hlf/chaincode/matriculation.md

```diff
@@ -166,3 +174,22 @@ This method adds a single entry to the list of semesters in the MatriculationDat
   "semesters": ["WS2019/20", "SS2020"]
 }
 ```
+
+## <a id="Checks" />Input Checks
+### <a id="matriculationDataChecks" />matriculationData
+- Checks, if parseable Json.
+- **enrollmentId**
+  - Check, if not null or an empty String.
+- **matriculationStatus**
+  - see [subjectMatriculation](#subjectMatriculationChecks)
+
+
+### <a id="subjectMatriculationChecks" />subjectMatriculation
+- Check, if not null or an empty list.
+- **fieldOfStudy**
+  - Check, if not null or an empty String.
+  - Check for duplicate entries.
+- **semesters**
+  - Check, if not null or an empty list.
+  - Check, if containing strings are in the format (WS\d{4}/\d{2}|SS\d{4})

```

Comment:
Do you only check the format? Is WS2020/23 valid for you?

Reply:
No, but apparently we forgot to document it. We also check, if the years contained in the winter semester are consecutive and in the right order. I added the docu on that.